modules/cloud4/modules/graph-development/pages/index.adoc

```diff
@@ -0,0 +1,83 @@
+= Graph Managment
```

Comment:
**Management** is misspelled. Because this is a heading, I don't know if you need to fix some references also.